src/lib/lightning/clightning/clightningService.ts

```diff
@@ -197,9 +199,100 @@ class CLightningService implements LightningService {
     );
   }
 
+  async addListenerToNode(node: LightningNode): Promise<void> {
+    console.log('addListenerToNode CLN on port: ', node.ports.rest);
```

Comment:
You should use `debug` from `electron-log` instead of `console.log`. Also would be good to add a comment stating that this is a no-op because polling is used for CLN nodes.